package.json

```diff
@@ -6,6 +6,7 @@
   "engines": {
     "node": ">= 16.0.0"
   },
+  "sideEffects": false,
   "repository": {
     "type": "git",
     "url": "http://github.com/TerriaJS/terriajs"
```

Comment:
There is a resolution of `terser-webpack-plugin` to version 4 a few lines below (introduced in commit 5b27cb2435). You might be able to drop the resolution, or at least update the resolution to version 5.